data_nutch_data_nci/conf/seeds.txt

```diff
@@ -1,24 +1,25 @@
 ## Main CGov Site to Start
-https://www.cancer.gov/sitemap.xml	nutch.fetchInterval=86400 
```

Comment:
Wouldn't you want to keep `sitemap.xml`?  The microsites (e.g. `/nano`) don't otherwise appear in this list.

Reply:
https://www.cancer.gov/sitemap.xml contains below, SWS doesn't need dictionaries etc.
<sitemap>
<loc>https://www.cancer.gov/sitemaps/pageinstructions.xml</loc>
<lastmod>2019-06-18T07:59:48.4052333-04:00</lastmod>
</sitemap>
<sitemap>
<loc>https://www.cancer.gov/sitemaps/dictionaries.xml</loc>
<lastmod>2019-06-18T07:59:48.4052333-04:00</lastmod>
</sitemap>
<sitemap>
<loc>https://www.cancer.gov/sitemaps/dynamiclistingpages.xml</loc>
<lastmod>2019-06-18T07:59:48.4052333-04:00</lastmod>
</sitemap>
<sitemap>
<loc>https://www.cancer.gov/sitemaps/clinicaltrials.xml</loc>
<lastmod>2019-06-18T07:59:48.4052333-04:00</lastmod>
</sitemap>

Reply:
That's what I'm getting at.  This doesn't have anything to point nutch to www.cancer.gov/nano, www.cancer.gov/pediatric-adult-rare-tumor, and the other microsites which are listed in the main sitemap.xml.

(On the other hand, the file **does** contain a reference to https://cancergenome.nih.gov/, and that site has been a redirect to https://www.cancer.gov/ccg/research/genome-sequencing/tcga for several years.)